Scripts/Build-Phases/sparkle-code-sign.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
This was inlined in the project file. I merely extracted it to a dedicated file for clarity.